libs/wingsdk/src/target-tf-gcp/app.ts

```diff
@@ -0,0 +1,84 @@
+import { GoogleProvider } from "@cdktf/provider-google/lib/provider";
+import { RandomProvider } from "@cdktf/provider-random/lib/provider";
+import { IConstruct } from "constructs";
+import { PolyconFactory } from "./factory";
+import { AppProps as CdktfAppProps, CdktfApp, IApp } from "../core";
+
+/**
+ * GCP App props.
+ */
+export interface AppProps extends CdktfAppProps {
+  /**
+   * The Google Cloud project ID.
+   */
+  readonly projectId: string;
+
+  /**
+   * The Google Cloud storage location, used for all storage resources.
+   * @see https://cloud.google.com/storage/docs/locations
+   */
+  readonly storageLocation: string;
```

Comment:
Just curious, do you imagine we will have a separate location for other resources types? It looks like other resources also take a location, but from what I see initially the location values are not uniform.

It looks like the storage locations could potentially be used with queues https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_tasks_queue

But then there is something like kms key rings which use their own location list (shown by `gcloud kms locations list`)
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/kms_key_ring

Yea not really a comment more an observation, interesting to see different dynamics in the clouds. 😅

Reply:
Yep, exactly. That's what I inferred from reading the GCP docs as well -  it seems like different resources have very different lists of regions. (I never thought I'd appreciate AWS regions before today 😅).

It's possible we can make an enum with a list of all of the available locations, and indicate which ones support which GCP resources etc., but I thought we can start with `storageLocation` and iterate from there.